book/02-git-basics/sections/recording-changes.asc

```diff
@@ -466,6 +466,8 @@ $ git commit -a -m 'added new benchmarks'
 ----
 
 Notice how you don't have to run `git add` on the `CONTRIBUTING.md` file in this case before you commit.
+But beware because you can commit files that you don't necessarily want to commit. Be diligent when you need to commit files, and most
+of the time you should stage and then commit you files.
```

Comment:
I like where this is going. How about something like this?

```
Notice how you don't have to run `git add` on the `CONTRIBUTING.md` file in this case before you commit.
That's because the `-a` flag includes all changed files.
This is convenient, but be careful; sometimes this flag will cause you to include unwanted changes.
```